packages/transform/src/index.ts

```diff
@@ -1,10 +1,200 @@
+// npx asc myTest.ts ./packages/assembly/index.ts --transform ./packages/transform/src/index.ts --textFile output.wat
+
 import {
+  BlockStatement,
   Transform,
+  Source,
+  Statement,
   Parser,
+  TernaryExpression,
+  IfStatement,
+  BinaryExpression,
+  Token,
+  SwitchCase,
+  CallExpression,
+  Expression,
+  NodeKind,
 } from "visitor-as/as";
 
+import { SimpleParser, BaseVisitor } from "visitor-as";
+import linecol from "line-column";
+// -- Imports
+class CoverTransform extends BaseVisitor {
+  private id: number = 0;
+  private linecol: any = 0;
+  private globalStatements: Statement[] = [];
+  public sources: Source[] = [];
+  private sourceId: number = 0;
+  // Declare properties.
+  visitBinaryExpression(expr: BinaryExpression): void {
+    const name = expr.range.source.normalizedPath;
+
+    if (expr.operator === Token.AMPERSAND_AMPERSAND) {
+      const leftExpression = expr.left;
```

Comment:
We want to cover the `right` expression, because the left one will always be reached.